File: Python/Petshop.py
class Petshop:
    def __init__(this, id=0, namaProduk="", hargaProduk=0, stokProduk=0):
        this.id = id
        this.namaProduk = namaProduk
        this.hargaProduk = hargaProduk
        this.stokProduk = stokProduk
        
    # Setter & Getter
    def get_id(this):
        return this.id
    
    def set_id(this, id):
        this.id = id
        
    def get_namaProduk(this):
        return this.namaProduk
    
    def set_namaProduk(this, namaProduk):
        this.namaProduk = namaProduk
        
    def get_hargaProduk(this):
        return this.hargaProduk
    
    def set_hargaProduk(this, hargaProduk):
        this.hargaProduk = hargaProduk
        
    def get_stokProduk(this):
        return this.stokProduk
    
    def set_stokProduk(this, stokProduk):
        this.stokProduk = stokProduk

File: CPP/PetShop.cpp
#include <iostream>
#include <iomanip>
using namespace std;

class PetShop
{
private:
    int id;
    string namaProduk;
    int hargaProduk;
    int stokProduk;

public:


    PetShop()
    {
    }


    PetShop(int id, string namaProduk, int hargaProduk, int stokProduk)
    {
        this->id = id;
        this->namaProduk = namaProduk;
        this->hargaProduk = hargaProduk;
        this->stokProduk = stokProduk;
    }
    

    // Getter & Setter
    int getId()
    {
        return this->id;
    }

    void setId(int id)
    {
        this->id = id;
    }

    string getNamaProduk()
    {
        return this->namaProduk;
    }

    void setNamaProduk(string namaProduk)
    {
        this->namaProduk = namaProduk;
    }

    int getHargaProduk()
    {
        return this->hargaProduk;
    }

    void setHargaProduk(int hargaProduk)
    {
        this->hargaProduk = hargaProduk;
    }

    int getStokProduk()
    {
        return this->stokProduk;
    }

    void setStokProduk(int stokProduk)
    {
        this->stokProduk = stokProduk;
    }

    // getJumlahProduk
    int getJumlahProduk()
    {
        return 0;
    }

    
    ~PetShop()
    {
    }
};
